modules/ROOT/pages/profile-creation.adoc

```diff
@@ -71,7 +66,7 @@ A toolbar is embedded to the editor, use it to:
 
 * *Add* a profile to this file of profiles: you can add a new profile or duplicate an existing one from your current project
 * *Save* this file of profiles *As* an other file of profiles (useful to duplicate the whole file)
-* *Deploy* this file of profiles (i.e all the profiles in this file)
+* *Deploy* this file of profiles (i.e all the profiles in this file) +
 ⚠  For _development_ purposes, it also deploys their referenced pages used as portal menu entries, but if those resources use REST API Extensions, you have to deploy those manually. In _production_ environment, you still need to import pages, and REST API Extensions manually *before* to import the file of profiles.
```

Comment:
*before* to import the file of profiles. => *before* importing profiles file.

Reply:
I modified it to "before importing profiles"

Reply:
Added